wgpu-hal/src/vulkan/device.rs

```diff
@@ -1516,6 +1517,7 @@ impl crate::Device<super::Api> for super::Device {
                     .shared
                     .workarounds
                     .contains(super::Workarounds::SEPARATE_ENTRY_POINTS)
+                // or if it has any overrides
```

Comment:
Is this a TODO comment?

Reply:
Ah yeah, I'll remove it. It slipped through.